source/blog/2017-11-17-release-notes.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: 🍠 November 2017 Release Notes - Version 17.11.0
+published: true
+post_author:
+  display_name: Moses Nakamura
+  twitter: mnnakamura
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+It's November, and you know what that means!  Library releases!!!  We're now publishing
+with sbt 1.0.x, and we now also publish sbt-scrooge-plugin for sbt 1.0.x.
```

Comment:
"..and the sbt-scrooge-plugin additionally for sbt 1.0.x" ?

just want to make it clear it's being publishing in addition for sbt 1.0.x as we still publish for sbt 0.13.x